handlers/schemas/UploadVOD.json

```diff
@@ -0,0 +1,66 @@
+{
```

Comment:
Does `gojsonschema` work with YAML? This'd be so much cleaner if so
https://gist.github.com/victorges/723fb5e751a57b5e309412faa7cd8eed

Reply:
There seems to be no straight forward way to make YAML schemas work. Any guidance would be helpful

Reply:
Ah, don't worry too much about this, just a nice to have! If you wanna give it a shot tho, you could try using a lib [like this](https://github.com/kubernetes-sigs/yaml#installation-and-usage) which provides a `yaml.YAMLToJSON` helper.

Reply:
Done

Reply:
Nice!